### ISSUE TITLE
显存不够问题

### ISSUE THREAD

**Issue by sunyilgdx** [ymcui/Chinese-BERT-wwm#41]
请问一下，如果机器显存只有32G，或者更小，batch_size无法设置过高的情况下有没有折中的解决方法，谢谢

**Comment by ymcui** [comment]
可以做梯度累计（gradient accumulation），这个在pytorch版的BERT中是支持的。tensorflow的话需要自己实现一下。
